kernel_gateway/gatewayapp.py

```diff
@@ -174,6 +174,16 @@ def allow_origin_default(self):
     def expose_headers_default(self):
         return os.getenv(self.expose_headers_env, '')
 
+    x_headers_env = 'KG_X_HEADERS'
+    x_headers = Unicode(config=True,
```

Comment:
Since it sounds like this is providing the identical functionality to that provided in Notebook, then I suggest we use the same option name (and variable): `trust_xheaders`.  This fits the boolean datatype better as well, since `x_headers` sounds more like a noun.

I also suggest using the `Bool` traitlet rather than `Unicode`.  This should eliminate the need for `strtobool` and import.

Reply:
The reason I used unicode and not Bool is because it is coming from an env variable and it didn't say in the documentation whether it would cast from a string or how. There is also a CBool, for casting bool, however it seems like it only works for integers. I'll look into it and update it with Bool or CBool if appropriate.

Agree about the name, I didn't know it existed in notebook prior to implementing.

Reply:
Sounds good.  Once the updates are in, we'll merge.  Thanks.